rollup.config.js

```diff
@@ -22,6 +31,7 @@ export default {
 	preserveEntrySignatures: 'strict',
 	inlineDynamicImports: true,
 	plugins: [
+    aliases,
```

Comment:
not sure what is going on with the indentation of this line.

Reply:
Seems that much of the file is using tabs. I will convert to spaces to be consistent with the rest of the project.